docs/sphinx/quickstart.rst

```diff
@@ -53,59 +54,107 @@ versions, is available at :ref:`prerequisites`.
 
-.. code-block:: sh
+#. First, set up a separate build directory to configure the project:
 
-    # In the HPX source directory
-    mkdir build && cd build
-    cmake -DCMAKE_INSTALL_PREFIX=/install/path ..
-    make install
+  .. code-block:: sh
 
-This will build the core |hpx| libraries and examples, and install them to your
-chosen location. If you want to install |hpx| to system folders, simply leave out
-the ``CMAKE_INSTALL_PREFIX`` option. This may take a while. To speed up the
-process, launch more jobs by passing the ``-jN`` option to ``make``.
+      cd ..
+      mkdir build && cd build
 
-.. tip::
+#. To configure the project you have the following options:
 
-   Do not set only ``-j`` (i.e. ``-j`` without an explicit number of jobs)
-   unless you have a lot of memory available on your machine.
+  * To build the core |hpx| libraries and examples, and install them to your
+  chosen location (suggested):
 
-.. tip::
+    .. code-block:: sh
+
+        cmake -DCMAKE_INSTALL_PREFIX=/install/path ..
+
+    .. tip::
+
+       If you want to change |cmake| variables for your build, it is usually a good
+       idea to start with a clean build directory to avoid configuration problems.
+       It is especially important that you use a clean build directory when changing
+       between ``Release`` and ``Debug`` modes.
+
+  * To install |hpx| to system folders, simply leave out the ``CMAKE_INSTALL_PREFIX`` option:
+
+    .. code-block:: sh
+
+        cmake ..
+
+  * If your dependencies are in custom locations, you may need to tell |cmake| where 
+  to find them by passing one or more of the following options to |cmake|:
+
+    .. code-block:: sh
+
+        -DBOOST_ROOT=/path/to/boost
+        -DHWLOC_ROOT=/path/to/hwloc
+        -DTCMALLOC_ROOT=/path/to/tcmalloc
+        -DJEMALLOC_ROOT=/path/to/jemalloc
+
+  * If you want to try |hpx| without using a custom allocator pass
+  ``-DHPX_WITH_MALLOC=system`` to |cmake|:
+
+    .. code-block:: sh 
+
+        cmake -DCMAKE_INSTALL_PREFIX=/install/path -DHPX_WITH_MALLOC=system ..
+
+    .. important::
+
+       If you are building |hpx| for a system with more than 64 processing units,
+       you must change the |cmake| variable ``HPX_WITH_MAX_CPU_COUNT`` (to a value at least as big as the
+       number of (virtual) cores on your system).
 
-   If you want to change |cmake| variables for your build, it is usually a good
-   idea to start with a clean build directory to avoid configuration problems.
-   It is especially important that you use a clean build directory when changing
-   between ``Release`` and ``Debug`` modes.
+#. Once |cmake| is complete, to build the project you run:
 
-If your dependencies are in custom locations, you may need to tell |cmake| where
-to find them by passing one or more of the following options to |cmake|:
+  .. code-block:: sh
+
+      make -jN # where N is the number of jobs
+      make install -jN # where N is the number of jobs 
```

Comment:
```suggestion
      make install
```
simply because there's no benefit to running this in parallel (it's a single job that doesn't run in parallel).

Reply:
or `make -jN install` which does both in one step?